config/olm/openshift/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../openshift
+  - ../../samples
+images:
+  - name: docker.io/dynatrace/dynatrace-operator:snapshot
+    newName: registry.connect.redhat.com/dynatrace/dynatrace-operator
+    newTag: v0.2.1
```

Comment:
For reference, I was creating temporary directories for custom Kubernetes configurations to avoid noising the commits, e.g.,

https://github.com/Dynatrace/dynatrace-operator/blob/c0501ad93fcb1bc2a71ec4435a2f0871438ff29f/Makefile#L72-L75